tools/image-tag

```diff
@@ -2,14 +2,20 @@
 
```

Comment:
nit: Can you set this to bash please? Refer #816

Reply:
With this PR, the script is exactly the same as [Loki's image-tag](https://github.com/grafana/loki/blob/main/tools/image-tag) except for the `#!/usr/bin/env bash`. Seems reasonable to also use bash then 🙂

Reply:
Addressed in b314bc7